test-stones-even-closer-together.sh

```diff
@@ -2,11 +2,19 @@
 
 # conveyor-speed=58mm/s
 
+start_hardware () {
+    echo "conveyor=running"
+    echo "compressor=start"
+}
+
+stop_hardware () {
+    echo "conveyor=stopped"
+    echo "compressor=stop"
+}
 run_blue_stone () { 
     echo "lightbarrier1=on"
-    sleep .45
+    sleep .1
```

Comment:
This was a commit mistake... Thanks for reviewing it!
As this script leads to a crash of the GUI simulation, I already fixed it, even though it would have been better to leave it there for future crash tests.
I will do revised test scripts versions anyway, to react on the latest farbsort-gui commit (stone destruction on lightbarrier deactivation). There I will put the old sleep values back in.
Expect next pull request within next 2 hours.